### ISSUE TITLE
Incompatible with @angular/material@7.0.0

### ISSUE THREAD

**Issue by snebjorn** [ncstate-sat/popover#139]
It appears that some of the peer dependencies need a version bump.

When running `ng update @angular/material` I get the following error

```shell
Package "@ncstate/sat-popover" has an incompatible peer dependency to "@angular/cdk" (requires "^6.0.0", would install "7.0.0").
Incompatible peer dependencies found. See above.
```

**Comment by DaGLiMiOuX** [comment]
+1

**Comment by BobaFett58** [comment]
+1

**Comment by g-schmitz** [comment]
+1

**Comment by willshowell** [comment]
Fixed with [v3.0.0](https://www.npmjs.com/package/@ncstate/sat-popover)!
